File: Week4/Training/MVVMArchitectureComp/app/src/main/java/com/example/admin/mvvmarchitecturecomp/data/remote/RepositoryModule/Repository.java
package com.example.admin.mvvmarchitecturecomp.data.remote.RepositoryModule;

import android.arch.lifecycle.MutableLiveData;

import com.example.admin.mvvmarchitecturecomp.data.remote.models.ApiList;

import java.util.List;

public interface Repository {

    MutableLiveData<List<ApiList>> weatherForecastLiveData = null;
    void getWeatherForecast();
}

File: Week5/Homework/CodeChallengeKotlin/app/src/main/java/com/example/admin/codechallengekotlin/data/repository/Repository.kt
package com.example.admin.codechallengekotlin.data.repository

import android.arch.lifecycle.MutableLiveData
import com.example.admin.codechallengekotlin.data.remote.models.MetaWeatherResponse

interface Repository {

    val weatherForecastLiveData: MutableLiveData<List<MetaWeatherResponse>>
    fun getWeatherForecast(woeId:String)
}